[2.9.0] Add known bugs section in Helm Charts page (#1384)
rancher/rancher-docs: versioned_docs/version-2.9/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md

## Limitations

- Dashboard apps or Rancher feature charts can't be installed using the Rancher CLI.

>   When determining the most recent version to display for the **Upgradable** column on the **Apps > Installed Apps** page, rather than only considering versions of the Helm chart from the repository it was installed from, Rancher considers versions of the Helm chart from all repositories on the cluster.

Reviewer: ```suggestion
-  When determining the most recent version to display for the **Upgradable** column on the **Apps > Installed Apps** page, rather than only considering versions of the Helm chart from the repository it was installed from, Rancher considers versions of the Helm chart from all repositories on the cluster.
```

> When determining the most recent version to display for the **Upgradable** column on the **Apps > Installed Apps** page, rather than only considering versions of the Helm chart from the repository it was installed from, Rancher considers versions of the Helm chart from all repositories on the cluster.
> 
> For example, you installed `cert-manager` v1.13.0 from repository A, where v1.14.0 is now the most recent version available. In this case, you expect **Upgradable** to display v1.14.0. However, if the cluster also has access to repository B where v1.15.0 of `cert-manager` is available, then **Upgradable** displays v1.15.0 even though the original installation used repository A.

Reviewer: ```suggestion
  When determining the most recent version to display for the **Upgradable** column on the **Apps > Installed Apps** page, rather than only considering versions of the Helm chart from the repository it was installed from, Rancher considers versions of the Helm chart from all repositories on the cluster.

  For example, you might install `cert-manager` v1.13.0 from repository A, where v1.14.0 is now the most recent version available. In this case, you expect **Upgradable** to display v1.14.0. However, if the cluster also has access to repository B where v1.15.0 of `cert-manager` is available, then **Upgradable** displays v1.15.0 even though the original installation used repository A.
```

Reviewer 2: I'm probably missing the point, but what's the purpose of changing to `might install` here? Doesn't that alter the meaning?

Reviewer: Trying to make it more speculative as it is an example. We could throw in an "if you installed" instead, or "For example, suppose you installed"

Reviewer 2: > We could throw in an "if you installed" instead, or "For example, suppose you installed"

I think these make more sense (or maybe just adjusting the tense on the "might" portion). For upgrade scenarios, the precondition is that X is already installed (with the version having significance in this case) so making that part certain doesn't seem right logically.

Also, looks like we forgot the `-` before `When` since that's the start of the list item.

>   For example, you might install `cert-manager` v1.13.0 from repository A, where v1.14.0 is now the most recent version available. In this case, you expect **Upgradable** to display v1.14.0. However, if the cluster also has access to repository B where v1.15.0 of `cert-manager` is available, then **Upgradable** displays v1.15.0 even though the original installation used repository A.

Reviewer: ```suggestion
  For example, suppose you install `cert-manager` v1.13.0 from repository A, where v1.14.0 is now the most recent version available. In this case, you expect **Upgradable** to display v1.14.0. However, if the cluster also has access to repository B where v1.15.0 of `cert-manager` is available, then **Upgradable** displays v1.15.0 even though the original installation used repository A.
```

>   For example, if you have a `cert-manager` v1.13.0 Helm chart installed, and you have two Helm repositories which both have cert-manager Helm charts, the first with the latest listed as v1.14.0 and the second with the latest listed as v1.15.0, then Rancher will only display v1.15.0 in the **Upgradeable** column, even though you might have installed `cert-manager` v1.13.0 from the first Helm repository.

Reviewer 2: ```suggestion
For example, you installed `cert-manager` v1.13.0 from repository A, where v1.14.0 is now the most recent version available on repository A. In this case, you expect **Upgradable** to display v1.14.0. However, if the cluster also has access to repository B where v1.15.0 of `cert-manager` is available, then **Upgradable** displays v1.15.0 even though the original installation used repository A.
```

## Limitations

Dashboard apps or Rancher feature charts can't be installed using the Rancher CLI.

## Known Bugs 

> As of now, Rancher UI displays the version in `Upgradable` column by combining the versions of the chart from all the Helm repositories installed in Rancher. 

Reviewer: ```suggestion
As of now, Rancher UI displays the version in `Upgradable` column by combining the versions of the chart from all the Helm repositories installed in Rancher. 
```

As of v2.9,0, or does this affect earlier Rancher versions?

Also: From the description that follows, I think "combining" might be too brief and suggest that it displays multiple values or aggregates them in some way. I think this needs to be rephrased. Something like, "only lists the most recent latest version from available repos, even if other available repos offer earlier latest versions of the same chart"

"Combining" here seems to mean "creates a list of available versions by combining values from multiple repositories"?

> As of now, Rancher UI displays the version in `Upgradable` column by combining the versions of the chart from all the Helm repositories installed in Rancher. 

Reviewer: ```suggestion
As of now, the **Apps > Installed Apps** page displays the version in the **Upgradable** column, by combining the versions of the chart from all the Helm repositories installed in Rancher. 
```

> As of now, the **Apps > Installed Apps** page displays the version in `Upgradable` column by combining the versions of the chart from all the Helm repositories installed in Rancher.

Reviewer 2: Has this always worked this way or is it a regression? If it's not a regression, maybe we can move it to the `Limitations` section and frame it as something that's hasn't been implemented yet as opposed to a bug.

For example, if you have cert-manager 1.13.0 helm chart installed and you have two Helm repositories which both have cert-manager helm charts, the first one has the latest version as 1.14.0 and the second Helm repository has latest version as 1.15.0, then Rancher will combine the versions of cert-manager helm charts from both Helm repositories and display the latest version i.e 1.15.0 to upgrade to in the `Installed Apps` page even though you might have installed cert-manager 1.13.0 from the first helm repository.

> This is being fixed and will soon be released in an upcoming patch version.
> 
> 

Reviewer 2: ```suggestion
```

We should avoid making commitments in the docs in case the fix isn't released at the time listed.

For our own references, is there an issue where the fix is being tracked so we know when to remove/update this section?
